packages/common/ide/types/Configuration.ts

```diff
@@ -8,6 +8,7 @@ export type CursorlessConfiguration = {
   wordSeparators: string[];
   experimental: { snippetsDir: string | undefined; hatStability: HatStability };
   decorationDebounceDelayMs: number;
+  debug: boolean;
```

Comment:
this setting will be exposed to lsp, as debug output will happen within lsp, because that's where tree-sitter will live